src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,109 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.interfaces.ProductDao;
+import com.internet.shop.exceptions.DataProcessingException;
+import com.internet.shop.library.Dao;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    @Override
+    public Product create(Product product) {
+        String query = "INSERT INTO products(product_name, product_price) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query,
+                    Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, product.getName());
+            statement.setDouble(2, product.getPrice());
+            statement.execute();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            while (resultSet.next()) {
+                product.setId(resultSet.getLong(1));
+            }
+            return product;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Failed to add the product", e);
+        }
+    }
+
+    @Override
+    public Optional<Product> get(Long id) {
+        String query = "SELECT product_id, product_name, product_price"
+                + " WHERE product_id = ? AND deleted = 0;";
```

Comment:
```suggestion
                + " WHERE product_id = ? AND deleted = FALSE;";
```
same with other queries

Reply:
changed